src/client.rs

```diff
@@ -99,63 +102,176 @@ impl Client {
         })
     }
 
-    /*
-    /// Send a Get message with a `DataIdentifier` to an `Authority`, signed with given keys.
-    pub fn send_get_request(&self,
-                            dst: Authority,
-                            data_id: DataIdentifier,
-                            message_id: MessageId)
-                            -> Result<(), InterfaceError> {
-        self.send_action(Request::Get(data_id, message_id), dst, CLIENT_GET_PRIORITY)
+    /// Fetches a latest version number of the provided MutableData
+    #[allow(unused)] // <-- TODO: remove this
+    pub fn get_mutable_data_version(&self,
```

Comment:
The api is not in sync with this : https://github.com/maidsafe/safe_core/pull/392/files#diff-b4e6c25f8e72ab235d15704d763ad8b3R123 - pls make it exactly have the same function signatures - else all code in safe_core will fail on switching from mock to actual routing.